src/applications/mhv-landing-page/tests/e2e/mhv-landing-page.cypress.spec.js

```diff
@@ -1,9 +1,25 @@
-import manifest from '../../manifest.json';
+import { appName, rootUrl } from '../../manifest.json';
+import user from '../fixtures/user.json';
+import { generateFeatureToggles } from '../../mocks/api/feature-toggles';
 
-describe(manifest.appName, () => {
-  it('Visit bad URL', () => {
-    cy.visit(`${manifest.rootUrl}/dummy`);
+describe(`${appName} - landing page`, () => {
+  beforeEach(() => {
+    cy.intercept('GET', '/v0/feature_toggles*', generateFeatureToggles()).as(
+      'featureToggles',
+    );
+  });
+
+  it('display the landing page when visiting root URL', () => {
+    cy.login(user);
+    cy.visit(rootUrl);
+    cy.injectAxeThenAxeCheck();
+    cy.findByRole('heading', { name: /^My HealtheVet$/i }).should.exist;
+    cy.findByRole('heading', { level: 2, name: /^Welcome/ }).should.exist;
```

Comment:
Note: Move the welcome message test here

Reply:
I think it makes sense to keep the tests for this component scoped to what is currently in production. In which case, the personalization feature flag should be off and `cy.findByRole('heading', { level: 2, name: /^Welcome/ }).should.not.exist;`.

Reply:
Yes. The file at `src/applications/mhv-landing-page/tests/e2e/personalization/welcome-message.cypress.spec.js` covers this.